src/sentry/utils/samples.py

```diff
@@ -207,7 +207,7 @@ def load_data(
             for key, entry in measurements.items():
                 if key in ["fp", "fcp", "lcp", "fid"]:
                     measurement_markers["mark.{}".format(key)] = {
-                        "value": data["start_timestamp"] + entry["value"] / 1000
+                        "value": round(data["start_timestamp"] + entry["value"] / 1000)
```

Comment:
```suggestion
                        "value": data["start_timestamp"] + entry["value"] // 1000
```

This will produce a rounded-down integer on both py2 and py3. Otherwise, this is float on py2.

Reply:
I'm also not the right person to judge whether ints are fine here, so will defer that to others.

Reply:
We need to preserve the fractional component.

```suggestion
                        "value": round(data["start_timestamp"] + entry["value"] / 1000, 3)
```

Reply:
Need to keep up to millisecond precision, so using `round(x, 3)` like @dashed suggested.